generators/server/templates/gradle/docker.gradle.ejs

```diff
@@ -0,0 +1,25 @@
+dockerfile {
+    baseImage = "openjdk:14-alpine"
```

Comment:
`baseImage` is using by default "openjdk:14-alpine", added here only as "documentation", so users can switch without digging into docs. Same for nativeImage below. Not tested yet, but if anyone isn't using metrics and doesn't have frontend, it should work(maybe).